shared/src/constants/constants.ts

```diff
@@ -7,6 +7,25 @@ export const NETWORK_NAMES: Record<number, string> = {
   42: "kovan",
 };
 
+export const FullVaultList = [
+  "ryvUSDC-ETH-P-THETA",
+  "rETH-THETA",
+  "rBTC-THETA",
+  "rUSDC-ETH-P-THETA",
+  "rUSDC-BTC-P-THETA",
+] as const;
+export type VaultOptions = typeof FullVaultList[number];
+const ProdExcludeVault: VaultOptions[] = ["rUSDC-BTC-P-THETA"];
```

Comment:
Let's exclude the vault from prod for now

Reply:
Yep done

Reply:
Yep done